teamcity/README.md

```diff
@@ -28,6 +28,10 @@ The TeamCity check is included in the [Datadog Agent][1] package, so you don't n
 ![Guest user settings][20]
 ![Assign Role][21]
 
+**Note:**
+- For basic HTTP authentication specify identified `username` and `password` in `teamcity.d/conf.yaml` at the `conf.d/` folder of your [Agent's configuration directory][3] 
+- `Access denied. Enable guest authentication or check user permissions.` would result in case of incorrect/insufficent user permission
```

Comment:
Do you think we could also mention the permissions related to `Agent Workload Statistics` at the same time? https://github.com/DataDog/architecture/blob/master/rfcs/agent-integrations/teamcity/teamcity.md#authentication

Reply:
Updated with the feedback from docs team.